requirements/base.in

```diff
@@ -1,10 +1,10 @@
 # Core requirements for using this application
 
 celery                                  # Run task workers in other locations
-Django>=1.8,<1.11                       # Web application framework
+Django>=1.8,<2.0                        # Web application framework
```

Comment:
I actually prefer to have it be 2.0 instead of 1.12, since 1.12 doesn't actually exist, and 2.0 is the next release version.